openmc/data/decay.py

```diff
@@ -570,3 +574,42 @@ def sources(self):
 
         self._sources = merged_sources
         return self._sources
+
+
+_DECAY_PHOTON_SOURCE = {}
+
+
+def decay_photon_source(nuclide: str) -> Univariate:
+    """Get photon source from the decay of a nuclide
+
+    This function relies on data stored in a depletion chain. Before calling it
+    for the first time, you need to ensure that a depletion chain has been
+    specified in openmc.config['chain_file'].
+
+    .. versionadded:: 0.14.0
+
+    Parameters
+    ----------
+    nuclide : str
+        Name of nuclide, e.g., 'Co58'
+
+    Returns
+    -------
+    openmc.stats.Univariate
+        Distribution of energies in [eV] of photons emitted from decay
+    """
+    if not _DECAY_PHOTON_SOURCE:
+        chain_file = openmc.config.get('chain_file')
```

Comment:
This code runs without producing errors and returns None when run on an older chain file

Perhaps we should introduce version numbers for the chain files like we have for the cross sections and track files

Reply:
```<schema version="0.1">``` at the top of each file would perhaps do the trick, then we check the version number in the python code

Reply:
Note that none of the existing format of the chain.xml file has changed, we've simply added new `<source>` elements. But, keeping a version there is a good idea in any case.

You bring up a good point that this will just result in the function returning `None` for all nuclides when an old chain file is specified. I'll add a warning in here if after loading data from the chain file, the `_DECAY_PHOTON_SOURCE` dictionary is still empty.

Reply:
That sounds good, I think it will save use a few discourse posts if the code can detect that no source info was found